### PR TITLE
Minor grammar, spelling, and localization fixes.

### PR DESCRIPTION
**Pull request by kperdue2002**
## About The Pull Request

Some Oddities have descriptions with poor grammar, misspellings, missing words, or poor localization. This fixes that. (no offence to whoever wrote them)

## Details

<details>
  <summary>Expand</summary>
Fixes minor grammar, spelling, and localization issues.
Also fixes the Obscure Report's broken icon.
</details>

### CODE REVIEW

File: code/game/objects/items/oddities.dm [hunk 1]
```diff
@@ -119,8 +119,8 @@
 
 /obj/item/weapon/oddity/common/paper_omega
 	name = "collection of obscure reports"
-	desc = "Even the authors seem to be rather skeptical about their findings. Yet they are not connected to each other, but results are similar."
-	icon_state = "paper_omega"
```
Comment (MadmanMartian) [inline code/game/objects/items/oddities.dm]: Find the icon state that it's supposed to be and set it to that

Comment (kperdue2002) [reply code/game/objects/items/oddities.dm]: sure, I'll dig. However, I dont know what the intended icon is. Obciously something named "paper omega" but it appears as a blank icon in game with no clickable icon outside of storage containers and the right-click menu

File: code/game/objects/items/oddities.dm [hunk 2]
```diff
@@ -80,7 +80,7 @@
 
 /obj/item/weapon/oddity/common/photo_coridor
 	name = "surreal maint photo"
-	desc = "The corridor in this photograph looks familiar, though something seems wrong about it; as if everything in it was replaced with an exact replica of itself."
+	desc = "The corridor in this photograph looks familiar, though something seems wrong about it; It's as if everything in it was replaced with an exact replica of itself."
```
Comment (MadmanMartian) [inline code/game/objects/items/oddities.dm]: But, a semi-colon doesn't mean you are starting a new sentence, it's for a pause longer than a comma. The previous itteration still made sense.

File: code/game/objects/items/oddities.dm [hunk 3]
```diff
@@ -158,7 +158,7 @@
 
 /obj/item/weapon/oddity/common/old_money
 	name = "old money"
-	desc = "It's not like organization that issued this exist now."
+	desc = "It's not like organization that issued this exists anymore."
```
Comment (MadmanMartian) [inline code/game/objects/items/oddities.dm]: "It's not like the organization that issued this exists anymore."

File: code/game/objects/items/oddities.dm [hunk 4]
```diff
@@ -196,7 +196,7 @@
 
 /obj/item/weapon/oddity/common/teddy
 	name = "teddy bear"
-	desc = "He will there for you, in tough times."
+	desc = "He will be there for you, even through tough times."
```
Comment (MadmanMartian) [inline code/game/objects/items/oddities.dm]: I feel there was more drama with the "in tough times".

In addition, isn't this a reference to the risk of rain item: https://riskofrain.fandom.com/wiki/Tough_Times

So, the "in tough times" would be fitting with the reference itself.

File: code/game/objects/items/oddities.dm (hunk 3 above)
Comment (kperdue2002) [reply code/game/objects/items/oddities.dm]: i missed the "the", oops

File: code/game/objects/items/oddities.dm (hunk 4 above)
Comment (kperdue2002) [reply code/game/objects/items/oddities.dm]: thank you for the feedback

**Comment by kperdue2002** [comment]
ill use the feedback and improve my PR, thanks for the comments :)

**Pull request merged**